### PR TITLE
feat(send-data): improve OpenTelemetry logs collection configurations

### PR DESCRIPTION
**Pull request by ajaiswals**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 1]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
 
-The Sumo Logic Distribution for OpenTelemetry Collector provides various receivers for log collection. This document describes the receivers most commonly used for logs: the [Filelog Receiver](#filelog-receiver) and [Windows Log Event Receiver](#windows-log-event-receiver).
+The Sumo Logic Distribution for OpenTelemetry Collector provides various receivers for log collection. This document describes the receivers most commonly used for logs:
+
+
+* [Filelog Receiver](#filelog-receiver)
+* [Windows Log Event Receiver](#windows-log-event-receiver)
+* [Syslog Receiver](#syslog-receiver)
+
+You can find the full list of receivers on our [Sumologic OpenTelemetry Collector page](https://github.com/SumoLogic/sumologic-otel-collector/tree/main#components).
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:15]:
```suggestion
You can find the full list of receivers on our [Sumo Logic OpenTelemetry Collector page](https://github.com/SumoLogic/sumologic-otel-collector/tree/main#components).
```

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 2]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
 
-You can find the full list of receivers on our [OpenTelemetry Collector GitHub page](https://github.com/SumoLogic/sumologic-otel-collector/tree/main#components).
 
 ## Filelog Receiver
 
-The Filelog Receiver tails and parses logs from files. The following is a basic configuration for the Filelog Receiver (collecting logs from a file), which you can place in the `conf.d` directory:
+Following configuration demonstrates : 
+ 1. Collect   : How to collect logs stored in /var/log/syslog and /var/log/audit/audit.log
+ 2. Transform : Set \_sourceCatetgory field to "application_logs_prod"
+ 3. Export    : Send data to authenticated Sumologic organization.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:23]:
```suggestion
Following configuration demonstrates: 
1. Collect. How to collect logs stored in /var/log/syslog and /var/log/audit/audit.log.
2. Transform. Set `_sourceCatetgory` field to `application_logs_prod`.
3. Export. Send data to authenticated Sumo Logic organization.
```

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 3]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
-`include:` lets the Filelog Receiver know where the log file is placed. Make sure that the collector has permissions to access the files; otherwise, it will not be collected.
 
-The `include_file_name: false` prevents the receiver from adding `log.file.name` attribute to the logs.
+### Configuration details
+
+ 1. Create a file in folder /etc/otelcol-sumo/conf.d  with name for your choice, sample_app_configuration.yaml.
+ 2. Paste the above content.
+ 3. Restart collector with following command:
+      - Linux : systemctl restart otelcol-sumo
+      - Windows : Restart-Service -Name OtelcolSumo
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:60]:
```suggestion
1. Create a file in folder /etc/otelcol-sumo/conf.d  with name for your choice, sample_app_configuration.yaml.
2. Paste the above content.
3. Restart collector with following command:
   - For Linux:
     ```
     systemctl restart otelcol-sumo
     ```
   - Windows:
     ```
     Restart-Service -Name OtelcolSumo
     ```
```

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 4]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
+ 
+* __exporters__: 
+  *  `sumologic` Sends data to registered Sumologic Organization.  Auto configured in sumologic.yaml during installation using script.
+ 
+* __service__: 
+  *  `logs/pipeline1:`  Pipeline glues together what data to collect(receiver), how to process the data(processors) and where to send logs(exporter).
 
-The remaining processors in pipeline are from `sumologic.yaml` file and should be applied for better performance of collector and use of Sumo Logic platform.
+
+
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:78]:
```suggestion
```

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 5]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
+  * `include:` Lets the Filelog Receiver know where the log file is placed. Make sure that the collector has permissions to access the files; otherwise, it will not be collected.
+  * `include_file_path_resolved: true` Adds a `log.file.path_resolved` attribute to the logs that contain the whole path of the file.
+  * `storage: file_storage:` Prevents the receiver from reading the same data over and over again on each otelcol restart. Extension defined by default in sumologi.yaml
+ 
+* __processors__: 
+  * `resource/1` Name of the processor that uses `attributes` to add \_sourceCategory field with value `application_logs_prod`
+ 
+* __exporters__: 
+  *  `sumologic` Sends data to registered Sumologic Organization.  Auto configured in sumologic.yaml during installation using script.
+
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:73]:
```suggestion
* __processors__: 
  * `resource/1` Name of the processor that uses `attributes` to add `_sourceCategory` field with value `application_logs_prod`.
* __exporters__: 
  *  `sumologic` Sends data to registered Sumologic Organization.  Auto-configured in sumologic.yaml during installation using script.
```

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 6]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
 
-To prevent the receiver from reading the same data over and over again on each otelcol restart, also add the [File Storage extension][filestorageextension_docs] which is already defined in `sumologic.yaml`, that will allow Filelog receiver to persist the current position in watched files between otelcol restarts. Here's an example of such a configuration:
+## Windows Log Event Receiver
 
+Windows Log Event Receiver reads and parses logs from windows event log API to collect local events as you see on Windows Event Viewer.
+
+Following configuration demonstrates:
+ 1. Collect : Collect Application, Security and System channels 
+ 2. Transform : Set \_sourceCatetgory field to "windows_event_log_prod" 
+ 3. Export : Send data to authenticated Sumologic organization
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:95]: See https://github.com/SumoLogic/sumologic-documentation/pull/2501/files#r1208088856 for formatting

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 7]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
+      exporters: 
+              sumologic
+  ```
+
+### Configuration details
+
+ 1. Create a file in folder /etc/otelcol-sumo/conf.d  with name for your choice, sample_syslog_configuration.yaml.
+ 2. Paste the above content.
+ 3. Restart collector with following command:
+      - Linux : systemctl restart otelcol-sumo
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:197]:
```suggestion
1. Create a file in folder /etc/otelcol-sumo/conf.d  with name for your choice, sample_syslog_configuration.yaml.
2. Paste the above content.
3. Restart collector with following command:
      - Linux: `systemctl restart otelcol-sumo`
```

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 8]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
-      - sumologic
+        - sumologic
 ```
+### Configuration details
 
-For more details, see the [Filelog Receiver documentation][filelogreceiver_readme].
+ 1. Create a file in folder /etc/otelcol-sumo/conf.d  with name sample_windows.yaml.
+ 2. Paste the above content.
+ 3. Restart collector with following command:
+      - Restart-Service -Name OtelcolSumo
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:132]:
```suggestion
1. Create a file in folder /etc/otelcol-sumo/conf.d  with name sample_windows.yaml.
2. Paste the above content.
3. Restart collector with following command: `Restart-Service -Name OtelcolSumo`.
```

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 9]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
-To prevent the receiver from reading the same data over and over again on each otelcol restart, also add the [File Storage extension][filestorageextension_docs] which is already defined in `sumologic.yaml`, that will allow Filelog receiver to persist the current position in watched files between otelcol restarts. Here's an example of such a configuration:
+## Windows Log Event Receiver
 
+Windows Log Event Receiver reads and parses logs from windows event log API to collect local events as you see on Windows Event Viewer.
+
+Following configuration demonstrates:
+ 1. Collect : Collect Application, Security and System channels 
+ 2. Transform : Set \_sourceCatetgory field to "windows_event_log_prod" 
+ 3. Export : Send data to authenticated Sumologic organization
+
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:96]:
```suggestion
Following configuration demonstrates:
1. Collect. Collect Application, Security and System channels. 
2. Transform. Set `_sourceCatetgory` field to "windows_event_log_prod".
3. Export. Send data to authenticated Sumo Logic organization.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
formatting - otherwise, content looks great. thank you for your help!

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 10]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
-      - filelog/custom_files
+        - filelog/1
       processors:
-      - memory_limiter
-      - groupbyattrs/file path resolved
-      - resourcedetection/system
-      - batch
+        - resource/1
+        - batch
+        - memory_limiter
```
Comment (astencel-sumo) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:48]: Memory limiter should be the first processor in the processors pipeline.

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 11]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
 ```yaml
 receivers:
-  filelog/custom_files:
-    include:
-    - /var/log/myservice/*.log
-    - /other/path/**/*.txt
-    - ./tmp/logs/*.log
-    include_file_name: false
-    include_file_path_resolved: true
+  filelog/custom_files:
```
Comment (astencel-sumo) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:27]: The trailing spaces should be removed.

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 12]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
 receivers:
-  filelog/custom_files:
-    include:
-    - /var/log/myservice/*.log
-    - /other/path/**/*.txt
-    - ./tmp/logs/*.log
-    include_file_name: false
-    include_file_path_resolved: true
+  filelog/custom_files:                                         
+    include_file_path: true
```
Comment (astencel-sumo) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:28]: Why changing this from `include_file_path_resolved: true` to `include_file_path: true`?
Why is the `include_file_name: false` being removed?

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 13]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
+      - key: _sourceCategory
+        value: application_logs_prod
+        action: insert
 service:
   pipelines:
-    logs/custom_files:
+    logs/pipeline1:
       receivers:
-      - filelog/custom_files
+        - filelog/1
```
Comment (astencel-sumo) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:44]: The name `filelog/1` is different from the `filelog/custom_files` defined above. The collector will fail to start with this config.

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 14]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
-      - log.file.path_resolved
+  resource/processor1:
+    attributes:                                       
+      - key: _sourceCategory
+        value: application_logs_prod
+        action: insert
 service:
   pipelines:
-    logs/custom_files:
+    logs/pipeline1:
```
Comment (astencel-sumo) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:42]: `pipeline` - Is it a name we want to use? Perhaps we want to use something more descriptive?

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 15]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
       receivers:
-      - filelog/custom_files
+        - filelog/1
       processors:
-      - memory_limiter
-      - groupbyattrs/file path resolved
-      - resourcedetection/system
-      - batch
+        - resource/1
```
Comment (astencel-sumo) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:46]: The name `resource/1` is different from the above defined `resource/processor`.

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 16]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
+    include:                                         
+      - /var/log/syslog
+      - /var/log/audit/audit.log
+     storage: file_storage
+     
 processors:
-  groupbyattrs/file path resolved:
-    keys:
-      - log.file.path_resolved
+  resource/processor1:
```
Comment (astencel-sumo) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:35]: Do we want to use this name `processor1`? I'd prefer something more descriptive.

**Review by astencel-sumo — DISMISSED** [review]

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 17]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
+Following configuration demonstrates:
+ 1. Collect : Collect syslog sent using udp protocol to 127.0.0.1 on port 514
+ 2. Transform : Set \_sourceCatetgory field to "syslog_event_log_prod" 
+ 3. Export : Send data to authenticated Sumologic organization
+
+```yaml
+
+receivers:
+  syslog:
+     udp:
```
Comment (astencel-sumo) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:169]:
```suggestion
    udp:
```

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 18]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
+    listen_address: "127.0.0.1:514"
```
Comment (astencel-sumo) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:170]:
```suggestion
      listen_address: "127.0.0.1:514"
```

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 19]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
+ 
+service:
+  pipelines:
+    logs/syslog:
+      receivers: 
+        - syslog
+      processors: 
+        - resource/syslog:
+      exporters: 
+              sumologic
```
Comment (astencel-sumo) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:189]:
```suggestion
        - sumologic
```

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 20]
```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
...
+        action: insert
+
+ 
+service:
+  pipelines:
+    logs/syslog:
+      receivers: 
+        - syslog
+      processors: 
+        - resource/syslog:
```
Comment (astencel-sumo) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:187]:
```suggestion
        - resource/syslog
```

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 21]
```diff
@@ -139,41 +260,8 @@ service:
       - sumologic
 ```
 
-## Windows Log Event Receiver
```
Comment (astencel-sumo) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:142]: Why is the section "Windows Log Event Receiver" removed?

Comment (astencel-sumo) [reply docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:142]: Oh, sorry, missed it - the section is just moved up.

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 22]
```diff
@@ -5,91 +5,203 @@ sidebar_label: Collect Logs
...
+  * `filelog/custom_files:` Unique name defined for the filelog receiver. You can add multiple file receivers as filelog/2 , filelog/prod etc.
+  * `include:` Lets the Filelog Receiver know where the log file is placed. Make sure that the collector has permissions to access the files; otherwise, it will not be collected.
+  * `include_file_path_resolved: true` Adds a `log.file.path_resolved` attribute to the logs that contain the whole path of the file.
+  * `storage: file_storage:` Prevents the receiver from reading the same data over and over again on each otelcol restart. Extension defined by default in sumologi.yaml
+* **processors**:
+  * `resource/1` Name of the processor that uses `attributes` to add `_sourceCategory` field with value `application_logs_prod`.
+* **exporters**:
+  * `sumologic` Sends data to registered Sumologic Organization. Auto-configured in sumologic.yaml during installation using script.
+* **service**:
+  *  `logs/pipeline1:` Pipeline glues together what data to collect(receiver), how to process the data(processors) and where to send logs(exporter).
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:74]:
```suggestion
  * `logs/pipeline1:` Pipeline glues together what data to collect(receiver), how to process the data(processors) and where to send logs(exporter).
```

**Comment by astencel-sumo** [comment]
@kimsauce I've reworked this PR after a chat with Arpit.

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 23]
```diff
@@ -5,145 +5,153 @@ sidebar_label: Collect Logs
...
-The Filelog Receiver tails and parses logs from files. The following is a basic configuration for the Filelog Receiver (collecting logs from a file), which you can place in the `conf.d` directory:
+:::tip Additional information
+See [Additional Configurations Reference](/docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference/) for more details about configuring the Sumo Logic OpenTelemetry Collector.
+:::
+
+## Collecting logs from local files
+
+Following configuration demonstrates:
+
+1. **Collect**: How to collect logs stored in /var/log/syslog and /var/log/audit/audit.log
```
Comment (perk-sumo) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:27]:
```suggestion
1. **Collect**: How to collect logs stored in `/var/log/syslog` and `/var/log/audit/audit.log`
```

**Review by perk-sumo — APPROVED** [review]
Two nits, other than that LGTM.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**